PROTOCOL.md

```diff
@@ -752,12 +863,12 @@ The requirements of the writers according to the protocol versions are summarize
 
 <br> | Requirements
 -|-
-Writer Version 2 | - Support [`delta.appendOnly`](#append-only-tables)<br>- Support [Column Invariants](#column-invariants)
+Writer Version 2 | - Support [Append-only Tables](#append-only-tables)<br>- Support [Column Invariants](#column-invariants)
 Writer Version 3 | Enforce:<br>- `delta.checkpoint.writeStatsAsJson`<br>- `delta.checkpoint.writeStatsAsStruct`<br>- [`CHECK` constraints](#check-constraints)
 Writer Version 4 | - Support [Change Data Feed](#add-cdc-file)<br>- Support [Generated Columns](#generated-columns)
 Writer Version 5 | Respect [Column Mapping](#column-mapping)
 Writer Version 6 | Support [Identity Columns](#identity-columns)
-Writer Version 7 | Respect [Deletion Vectors](#deletion-vectors)
+Writer Version 7 | Support [Table Features](#table-features) for writers
```

Comment:
This list is super inconsistent. Enforce, Support, Respect? Maybe we should standardize on "Respect".

```suggestion
Writer Version 7 | Respect [Table Features](#table-features) for writers
```